README.md

```diff
@@ -289,7 +289,7 @@ yo chisel:page "Home" "About Us" "Contact Us" "News"
 When you have the basic setup done, you can start development. To re-compile Twig, SCSS and JavaScript files in real time you can use default task. Type
 
 ```bash
-gulp
+npm run dev
```

Comment:
I can change the blog post if `npm start` is more common (for sure, it's shorter so I like it more too)